docs/faq.md

```diff
@@ -173,3 +173,32 @@ Tilt reads the same `KUBECONFIG` environment variable that `kubectl` uses. This
 sets the path to a Kubernetes config.
 
 <script src="/assets/js/links.js" async></script>
+
+### Q: How do I set up Tilt to use MicroK8s?
+
+If you're using Linux, [microk8s](https://microk8s.io/)
+is a fast, low-overhead way to run Kubernetes locally for development.
+
+To install it, run
+
+```
+$ sudo snap install microk8s --channel=1.13/stable --classic
+```
+
+To add microk8s to the list of clusters you can deploy to with `kubectl`, run
+
+```
+$ microk8s.kubectl config view --raw > $HOME/.kube/config
```

Comment:
Does this always work? My understanding is that sometimes the microk8s config changes when you restart it?

Reply:
Resolved offline. It might be worth doing a blog post on all the ways to set kubecon fig context